install/kubernetes/helm/istio/charts/pilot/templates/gateway.yaml

```diff
@@ -1,3 +1,31 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: istio-autogenerated-k8s-ingress
+  namespace: istio-system
+spec:
+  selector:
+    istio: {{ .Values.global.ingressSelector }}
+  servers:
+  - port:
+      number: 80
+      protocol: HTTP
+      name: http
+    hosts:
+    - "*"
+{{ if .Values.global.ingressHttps }}
+  - port:
+      number: 443
+      protocol: HTTPS
+      name: https-default
```

Comment:
if there is a chance that thsi gateway is bound to same workload as istio-ingressgateway, then name this as something unique.. like http-istio-reserved-port-blah

Reply:
as long as they have host names and use distinct port names, they would get separate RDS

Reply:
because we use SNI match to drop into a dedicated filter chain for the gateway